proto/pki/tx.proto

```diff
@@ -35,6 +35,7 @@ message MsgApproveAddX509RootCert {
   string subjectKeyId = 3 [(gogoproto.moretags) = "validate:\"required,max=256\""];
   string info = 4 [(gogoproto.moretags) = "validate:\"max=4096\""];
   int64 time = 5;
+  string subjectAsText = 6 [(gogoproto.moretags) = "validate:\"required,max=1024\""];
```

Comment:
You should not add `SubjectAsText` to transactions. Otherwise it becomes mandatory field for transaction

Reply:
Done! ✅